src/MizarData/include/MizarData/SamplingWithFrameTrackComparisonReport.h

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef MIZAR_DATA_SAMPLING_WITH_FRAME_TRACK_COMPARISON_REPORT_H_
+#define MIZAR_DATA_SAMPLING_WITH_FRAME_TRACK_COMPARISON_REPORT_H_
+
+#include <absl/container/flat_hash_map.h>
+#include <absl/container/flat_hash_set.h>
+#include <stdint.h>
+
+#include <cstdint>
+
+#include "MizarData/SampledFunctionId.h"
+#include "OrbitBase/Logging.h"
+
+namespace orbit_mizar_data {
+
+struct HalfOfSamplingWithFrameTrackReportConfig {
```

Comment:
I don't understand the name. Where does the frame track come in? Could you also add a comment explaining the purpose of the type?